net/munge/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        dun munge 0.5.12 munge-
+categories          net security
+license             GPL-3
+maintainers         kornel.us:karl
+description         Creating and validating credentials in HPC clusters.
+long_description    MUNGE (MUNGE Uid 'N' Gid Emporium) is an authentication \
+                    service for creating and validating credentials.  It is \
+                    designed to be highly scalable for use in an HPC cluster \
+                    environment.  It allows a process to authenticate the UID \
+                    and GID of another local or remote process within a group \
+                    of hosts having common users and groups.  These hosts form \
+                    a security realm that is defined by a shared cryptographic \
+                    key.  Clients within this security realm can create and \
+                    validate credentials without the use of root privileges, \
+                    reserved ports, or platform-specific methods.
+homepage            https://dun.github.io/munge/
+
+platforms           darwin
+checksums           rmd160 0bc366a2d0a49875e7cf35ea2aebeef33490ed0c \
+                    sha256 23585c1da3f4ea7c2882511c0a08220a2be13d9c03e54486bb8546791fa6c89b
+
+# We can either use OpenSSL or libgcrypt.  Let's default to OpenSSL.
+depends_lib-append  port:openssl
+use_configure       yes
+configure.args      --with-crypto-lib=openssl \
+                    --with-openssl-prefix=${prefix}
+
+# Allow using libgcrypt instead of OpenSSL.
+variant libgcrypt description {Use libgcrypt for cryptograpic routines} {
+    depends_lib-delete      port:openssl
+    depends_lib-append      port:libgcrypt
+    configure.args-delete   --with-crypto-lib=openssl \
+                            --with-openssl-prefix=${prefix}
+    configure.args-append   --with-crypto-lib=libgcrypt \
+                            --with-libgcrypt-prefix=${prefix}
+}
+
+# Allow doing testing
+test.run            yes
+test.target         check
+
+# When activating, make sure certain things are set
+post-activate {
+    # Create directories for config and for munge runtime
+    xinstall -m 700 -d ${prefix}/etc/munge
+    xinstall -m 711 -d ${prefix}/var/lib/munge
+    xinstall -m 755 -d ${prefix}/var/run/munge
```

Comment:
I didn't notice that functionality.  I'll move the installs to the destroot section!